### ISSUE TITLE
Skins cannot retrieve percentage of tvshow watched

### ISSUE THREAD

**Issue by jurialmunkey** [xbmc/xbmc#21543]
### Describe the bug
Skins are unable to display a progress control to provide a visual indication of the percentage of episodes watched in a tvshow or season as there is no appropriate infolabel for this data.

Currently, skins can only display the _number_ of unwatched or watched episodes. As a result, the user must read two different points of data and then do math to figure out how much of an in-progress tvshow that they have watched.

Visual indications would improve the user experience by allowing the user to see this progress information at a glance.

## Expected Behavior
Skins are able to display a visual indication of watched progress for tvshows and seasons via an infolabel.

## Actual Behavior
There is no 0-100 infolabel for this data.

## Possible Fix
A new 0-100 progress infolabel which contains `int(watchedepisodes / totalepisodes * 100)`

### Additional context
See the below screenshot where I have mocked up this behaviour using Arctic Horizon 2 and TMDb Helper. I am outputting this data to a property `ListItem.Property(WatchedProgress)` and then reading it in the skin. 

You should be able to see from this live mock-up how useful this progress indicator is. I have 11 episodes left to watch for both Fresh Off the Boat and Dark. Because FotB has 116 episodes but Dark has only 26, you can see the significantly different progress despite both having 11 episodes left. I'm only a little over half way through my re-watch of Dark but I've almost finished FotB.

![screenshot00112](https://user-images.githubusercontent.com/4618719/173222382-4064747f-8a76-4aa7-956b-dfc71e0767a9.png)

**Comment by jurialmunkey** [comment]
Ugh, didn't mean to close this one.

I know this is a bit borderline between bug and feature request. Happy to move to the forum if that would make more sense.

**Comment by DaVukovic** [comment]
Yeah, this sounds more like a feature request. But as you are a team member, I guess it's fine to leave it open. 

If you are capable of, it would be nice to see a PR ;)
